File: Chapter five/Java/StateCode.java
import java.util.Scanner;
public class StateCode{
public static void main(String[] args){
Scanner input = new Scanner(System.in);

System.out.println("Enter state code in Capital letter: ");
String code = input.nextLine();


System.out.println(checkStateCode(code));
}

public static String checkStateCode(String code){
if(code.equals("CT") || code.equals("MA") || code.equals("ME") || code.equals("NH") || code.equals("NJ") || code.equals("NY") || code.equals("PA") || code.equals("VT")){
return "State code is valid!!!";
}else
return "State code is incorrect!!! ";

}

}

File: Chapter five/Python/state_code.py
def check_state_code():
    code = input("Enter state code in Capital letter: ").upper()

    if code == "CT"  or code =="MA" or code == "ME" or code == "NH" or code == "NJ" or code == "NY" or code == "PA" or code == "VT":
        return "State code is valid!!!"
    return "State code is incorrect!!! "

print(check_state_code())

